continuous_integration/install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Maybe not in this PR, but `setup-miniconda` or `setup-micromamba` will be needed because on some macOS ARM there is no conda in the runner. I don't remember the details but https://github.com/actions/runner-images/issues/9262#issuecomment-1918858684 has more details.

Reply:
I took inspiration from the joblib's config. But there's a setup-miniconda in the test.yml config if that's what you're referring to

Reply:
I read too fast then and was mixed up between different things :sweat_smile: ...

Unrelated: I am pretty sure that libmamba is the default now, even more so because the setup-miniconda has `auto-update: true` but :shrug: